2021/spring/Sterne-MW@11/batsal-ghimire.md

```diff
@@ -0,0 +1,22 @@
+## My name is
+# BATSAL GHIMIRE
+#### and I am a human
+## being.

```

Comment:
While this shows some of your skills in using Markdown it negatively affects readability. Perhaps using headings only where necessary may make it more readable e.g. having a heading 'Short bio' and then in the regular text, you could introduce yourself.